src/libcore/num/mod.rs

```diff
@@ -416,6 +416,13 @@ macro_rules! int_impl {
 
         /// Saturating integer addition. Computes `self + other`, saturating at
         /// the numeric bounds instead of overflowing.
+        ///
+        /// # Examples
+        ///
```

Comment:
I've been adding `Basic usage:` before the first example block, can you do that as well, for consistency?